static/js/server_events.js

```diff
@@ -217,6 +217,9 @@ function dispatch_normal_event(event) {
             // than requiring a reload or page resize.
             page_params.left_side_userlist = event.setting;
         }
+        if (event.setting_name === 'emoji_alt_code') {
+            page_params.emoji_alt_code = event.setting;
+        }
```

Comment:
So, seeing this inspired me to fix that TODO above just now; we can remove the requirement to reload the browser and just have it magically live-update by doing the same thing as 9b11993fa74159cf520d4fb1aab648eef28237e3 in this section.

Also, while developing that, I noticed that you should update the tests in `frontend_tests/node_tests/dispatch.js` (in the `test-js-with-node` suite)